pkg/common/constants.go

```diff
@@ -108,17 +108,18 @@ const (
 )
 
 const (
-	RootDirPath            = "/"
-	DefaultImagePullPolicy = "IfNotPresent"
-	MyPodNamespace         = "MY_POD_NAMESPACE"
-	True                   = "true"
-	False                  = "false"
-	inject                 = ".fluid.io/inject"
-	injectSidecar          = ".sidecar" + inject
-	InjectServerless       = "serverless" + inject
-	InjectFuseSidecar      = "fuse" + injectSidecar
-	InjectCacheDir         = "cachedir" + injectSidecar
-	InjectWorkerSidecar    = "worker" + injectSidecar
-	InjectSidecarDone      = "done" + injectSidecar
-	App                    = "app"
+	RootDirPath             = "/"
+	DefaultImagePullPolicy  = "IfNotPresent"
+	MyPodNamespace          = "MY_POD_NAMESPACE"
+	True                    = "true"
+	False                   = "false"
+	inject                  = ".fluid.io/inject"
+	injectSidecar           = ".sidecar" + inject
+	InjectServerless        = "serverless" + inject
+	InjectFuseSidecar       = "fuse" + injectSidecar
+	InjectVirtualDevEnabled = "virtdev" + injectSidecar
```

Comment:
How about changing `virtdev` to `unprivileged`?

Reply:
Done